stm32-modules/include/thermocycler-refresh/thermocycler-refresh/lid_heater_task.hpp

```diff
@@ -61,6 +61,8 @@ requires MessageQueue<QueueImpl<Message>, Message>
 class LidHeaterTask {
   public:
     using Queue = QueueImpl<Message>;
+    using Milliseconds = uint32_t;
```

Comment:
https://en.cppreference.com/w/cpp/chrono/duration ? would understand if you don't want to